public/css/footer.css

```diff
@@ -0,0 +1,40 @@
+footer {
+  background-color: lightslategray;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+#footerGrid {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  justify-content: center; 
+}
+
+.footerLink {
+  height: 3vh;
```

Comment:
This is also good to have in an _:root_ css-file.
This makes it easier and faster for us to change css.